.github/workflows/golangci-lint.yaml

```
@@ -17,15 +17,15 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-go@v4
         with:
-          go-version: '1.21'
+          go-version: '1.23'
           cache: false
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
```

actionlint (1.7.7)

the runner of "golangci/golangci-lint-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)